wgengine/watchdog.go

```diff
@@ -60,6 +60,11 @@ type watchdogEngine struct {
 	inFlightCtr uint64
 }
 
+// inDebugger returns true if the process has $TS_DEBUGGER set. This should be
+// used to avoid terminating the process when operations take an unusual amount
+// of time, as would be caused by a debugger breakpoint.
+var inDebugger = envknob.RegisterBool("TS_DEBUGGER")
```

Comment:
we already have `TS_DEBUG_DISABLE_WATCHDOG`. Just use that instead? Or is there a difference?